algotaurus.py

```diff
@@ -565,15 +600,25 @@ def __init__(self, size=15, lines=30):
         self.root.update()
         w, h = self.root.winfo_screenwidth(), self.root.winfo_screenheight()
         winw, winh = self.root.winfo_width(), self.root.winfo_height()
-        self.padding = (winw-self.x*(self.size-1), winh-self.y*(self.size-1))
+        self.padding = (winw - self.maze_rows * (self.size - 1), winh - self.maze_columns * (self.size - 1))
         size = tuple(int(numb)+30 for numb in self.root.geometry().split('+')[0].split('x'))
         x = w/2 - size[0]/2
         y = h/2 - size[1]/2
-        self.root.geometry("%dx%d+%d+%d" % (size + (x, y)))
+        self.root.geometry("%dx%d+%d+%d" % (size + (x, y - 50)))
         self.root.minsize(winw, winh)
+        self.root.focus_force()
+        self.textPad.focus_set()
         self.root.mainloop()
 
     # Building menu and coder options
+    def change_language(self, event=None):
+        if language != self.lang_value.get():
+            cfgfile = open(dirs.user_config_dir+'/algotaurus.ini', 'w')
+            config.set('settings', 'language', self.lang_value.get())
+            config.write(cfgfile)
+            self.code = self.textPad.get('1.0', 'end'+'-1c')
+            self.root.destroy()  
+        
```

Comment:
Language change command: changes the configuration file and reloads the mainloop